docs/api-docs/storefront/graphql/graphql-api-overview.md

```diff
@@ -313,6 +215,97 @@ In addition to using `fetch()`, there's a other ways to query the API:
 </div>
 </div>
 
+<a id="accessing-the-graphql-playground" class="devdocsAnchor"></a>
+
+## Accessing the GraphQL Playground
+
+To access the GraphQL Storefront API Playground<sup>1</sup> and documentation:
+
+1. Login to a BigCommerce store enrolled in the beta
+2. Navigate to **Advanced Settings** > **Storefront API Playground**<sup>2</sup>
+
+The GraphQL Storefront API Playground will be opened:
+
+![GraphQL Storefront API Playground](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/graphql-storefront-api-playground.png "GraphQL Storefront API Playground")
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+    
+<!-- theme: info -->
+
+### Note
+
+> 1. GraphQL Playground is a GraphQL IDE built on Electron. For more information, see [GraphQL Playground](https://electronjs.org/apps/graphql-playground) on [electrongjs.org](https://electronjs.org)
+
+> 2. If the **Storefront API Playground** link is not visible, the store is not enrolled in the Beta program. To enroll, [contact support](https://support.bigcommerce.com/SubmitCase) (all stores using Stencil are now eligible).
+
+</div> 
+</div>
+</div>
+
+<a id="using-the-graphql-playground" class="devdocsAnchor"></a>
+
+## Using the GraphQL Playground
+
+To use the request runner, input queries on the left side and then click the play button. Query results will be displayed on the the right side:
+
+![GraphQL Playground Query](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/graphql-storefront-api-playground2.png "GraphQL Playground Query")
+
+Here's a sample Query to get you started:
+
+```javascript
+query MyFirstQuery {
+  site {
+    settings {
+      storeName
+    }
+    products {
+      edges {
+        node {
+          name
+          sku
+          prices {
+            retailPrice {
+              value
+              currencyCode
+            }
+            price {
+              value
+              currencyCode
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+To explore the storefront GraphQL schema, checkout the Docs and Schema tabs on the right:
+
+![GraphQL Playground Docs](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/graphql-playground-docs.png "GraphQL Playground Docs")
+
+Click changelog in the top right to view a list of recent changes to the storefront API:
+
+![GraphQL Playground Changelog](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/graphql-playground-changelog.png "GraphQL Playground Changelog")
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+<!-- theme: info -->
+
+### Note
+> * The changelog is updated with each deployment. 
+> * Additive Changes are possible during beta, so we recommend checking for changes frequently

```

Comment:
This line should end in a period since the preceding line did as well.

Regarding the whole document, there's inconsistency in whether list objects end in punctuation or not. For now, I would recommend sticking to one convention.